core-web/libs/dotcms-models/src/lib/dot-categories.model.ts

```diff
@@ -29,4 +29,9 @@ export interface DotCategory {
     parentPermissionable?: {
         hostname: string;
     };
+    parentList?: Array<{
```

Comment:
@oidacra Should this be a separate model? something like that:

```ts
type Parent = {
        categoryName: string;
        key: string;
        inode: string;
 } 
```

Reply:
moved